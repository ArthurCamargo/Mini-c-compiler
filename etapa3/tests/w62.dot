digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 [label="a"];
n3;
n3 -> n4;
n3 -> n5;
n4;
n4 [label="b"];
n5;
n5 -> n6;
n5 -> n7;
n6;
n6 [label="c"];
n7;
n7 [label="d"];
n5 [label="*"];
n3 [label="*"];
n1 [label="="];
n0 [label="f"];

}