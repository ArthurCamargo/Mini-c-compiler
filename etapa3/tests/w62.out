0x5597c7779350
0x5597c7779350, 0x5597c77797c0
0x5597c77797c0
0x5597c77797c0, 0x5597c7779580
0x5597c77797c0, 0x5597c7779770
0x5597c7779580
0x5597c7779580 [label="a"];
0x5597c7779770
0x5597c7779770, 0x5597c77795f0
0x5597c7779770, 0x5597c7779720
0x5597c77795f0
0x5597c77795f0 [label="b"];
0x5597c7779720
0x5597c7779720, 0x5597c7779660
0x5597c7779720, 0x5597c77796d0
0x5597c7779660
0x5597c7779660 [label="c"];
0x5597c77796d0
0x5597c77796d0 [label="d"];
0x5597c7779720 [label="*"];
0x5597c7779770 [label="*"];
0x5597c77797c0 [label="="];
0x5597c7779350 [label="f"];
